kroma-validator/l2_output_submitter.go

```diff
@@ -40,27 +43,40 @@ type L2OutputSubmitter struct {
 	log  log.Logger
 	metr metrics.Metricer
 
-	l2ooContract    *bindings.L2OutputOracleCaller
-	l2ooABI         *abi.ABI
-	valpoolContract *bindings.ValidatorPoolCaller
+	l2ooContract       *bindings.L2OutputOracle
+	l2ooABI            *abi.ABI
+	valpoolContract    *bindings.ValidatorPoolCaller
+	valManagerContract *bindings.ValidatorManagerCaller
+	valManagerAbi      *abi.ABI
 
 	singleRoundInterval *big.Int
 	l2BlockTime         *big.Int
 	requiredBondAmount  *big.Int
 
-	submitChan chan struct{}
+	isValManagerEnabled bool
+	terminationIndex    *big.Int
+
+	outputSubmittedSub ethereum.Subscription
+
+	submitChan                 chan struct{}
+	l2OutputSubmittedEventChan chan *bindings.L2OutputOracleOutputSubmitted
```

Comment:
Same here as challenger. I think it's unnecessary to update `isValManagerEnabled` with subscription. Just compare output index with `terminationIndex`.
```suggestion
	terminationIndex *big.Int

	submitChan chan struct{}
```